scripts/regression.py

```diff
@@ -19,9 +18,16 @@
 import MySQLdb
 
 
-def getoutput(cmd):
-    sp = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
-    return sp.communicate()[0].strip()
+def getstatusoutput(cmd):
+    """Return (status, output) of executing cmd in a shell."""
+    pipe = os.popen('{ ' + cmd + '; } 2>&1', 'r')
+    text = pipe.read()
+    sts = pipe.close()
+    if sts is None:
+        sts = 0
+    if text[-1:] == '\n':
+        text = text[:-1]
+    return sts, text
```

Comment:
If no use for the status, just define with getoutput and only return output is ok.

Reply:
Updated